### PR TITLE
Release v16.8.0

### PR DESCRIPTION
**Pull request by alex-ju**
## 16.8.0
* Add date input component based on GOV.UK Frontend (PR #792)
* Add cookie banner component and migrate the remaining [govuk-template.js](https://github.com/alphagov/govuk_template/blob/master/source/assets/javascripts/govuk-template.js) scripts from [govuk_template](https://github.com/alphagov/govuk_template/) (PR #795)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 16.8.0
 * Add date input component based on GOV.UK Frontend (PR #792)
```
Comment (andysellick) [inline CHANGELOG.md]: Nitpick: we seem to have lost the newline after the release heading for this and the last release.

Comment (alex-ju) [reply CHANGELOG.md]: Good catch, must have got lost in rebase. Updated.

**Comment by alex-ju** [comment]
@andysellick we usually run `bundle` when we do a release and (as far as I understand) one of the desired effects of it is to make sure we're updating the dependency tree - and that's what I've got :)

**Comment by andysellick** [comment]
@alex-ju sorry, I wasn't aware that was the procedure. No worries then.

**Comment by alex-ju** [comment]
@andysellick: it seems like we don't have the release process documented anywhere, which is a common question about this repo, so I'll make a note to add one.

**Pull request merged**